test/connect_test.js

```diff
@@ -242,6 +242,32 @@ exports.connect = {
       });
     });
   },
+  livereload_general: function(test) {
+    test.expect(2);
+    get({
+      hostname: 'localhost',
+      port: 8019,
+      path: '/livereload.html',
+      headers: {
+        accept: 'text/html'
+      }
+    }, function(res, body) {
+      test.ok(body.indexOf('https://example.org:54321/livereload.js?snipver=1') !== -1, 'Should contain livereload snippet with configured src.');
```

Comment:
Maybe just use `body.includes`?

Reply:
Just for curiosity: You wanted this change - why are you reverting it back now?

Reply:
Because I thought wrongfully we had switched to ES6 :P